### PR TITLE
Add a hint to avoid misunderstandings with regard to CLA signature in…

### PR DESCRIPTION
**Pull request by OSevangelist**
… article 22 of the Bylaws of the OCA. As discussed with @pedrobaeza  during the OCA Code Sprint 2018. That addition should clarify the contributor onboarding procedure.

### CODE REVIEW

File: Bylaws.rst [hunk 1]
```diff
@@ -201,6 +201,8 @@ Article 22
 
 OCA whishes that all contributors of ideas, code, or documentation to the Association projects complete, sign, and submit (via postal mail, fax or email) an Individual Contributor License Agreement (CLA). The purpose of this agreement is to clearly define the terms under which intellectual property has been contributed to the Association and thereby allow the Association to defend the project should there be a legal dispute regarding the software at some future time. A signed CLA is required for every Delegate member and before an individual is given commit rights to an OCA project.
 
+**Note** The CLA signature process is a bit different from the process applied by Odoo S.A. The OCA always requires written and signed documents and the OCA-Bot, upon commit, checks against a CLA flag in the backend of the OCA Odoo instance whether somebody has already signed the ICLA or is covered by an ECLA.
```
Comment (pedrobaeza) [inline Bylaws.rst]: I would also add that this CLA is different from Odoo one.

Comment (OSevangelist) [reply Bylaws.rst]: Ok i am gonna apply that right now

**Comment by OSevangelist** [comment]
@pedrobaeza done as requested

**Comment by dreispt** [comment]
Thank you Frederick for suggesting an improvement.
I think the actual text of the Bylaws should stay *ipsis verbis* with what was last approved by a general Assembly.

Clarifications on the actual OCA processes or application of the bylaws are better in a separate document.
At best it would be a footnote to the bylaws, but this case is not a clarification, but the description of an implemented process.

How about creating a "How do CLAs work?" page?

**Comment by OSevangelist** [comment]
@dreispt i thought of this as well but the problem is that this "difference" caused for a lot of uncertainty within the mini sprinters in Berlin. I simply fear that if we don' t put that into the document (at least in some way, i.e. maybe a link) it is very likely that others do the same mistake. And that is actually the entry point to contribute. So i am absolutely open for any solution that clarifies the matter buy keeping the Bylaws "ipsis verbis" would'nt maybe do the trick (unless we make sure that everywhere they are being linked my or a similar NOTE is beeing put)

**Comment by OSevangelist** [comment]
What do you think @elicoidal @jgrandguillaume ?

**Comment by OSevangelist** [comment]
> I won't oppose a footnote, with a reference/link to it just after the Bylaw clause.
> 

Ok i will propose a correction to let later this weekend or early next week

> I'm curious about the uncertainty: are the sprinters aware that the OCA is not Odoo SA?
Yes, i think so, but even me (even if i could know better by simple logical reasoning) was very confused as we pull all ECLA and ICLA into the doc folders of OCB and for any reason i though that would
do the trick also to get the OCA bot being green ;-)... Foolish i am ;-)

> Being different organizations, it is natural (to me) that they can have different CLA processes.
No doubt, yes!

> Probably the Bylaws shouldn't be the the first place to go when looking for information on how the bylaws work. An explanatory page would be a starting point. Probably it should even be printed together along with the actual CLA page, for perfect clarity.

Yes the latter is sound. Let's do it that way. A small explanatory note should go to 

https://odoo-community.org/page/Contribute

As i said i will work something out that clarifies the matter

**Comment by pedrobaeza** [comment]
@dreispt the confusion comes from OCB brances containing Odoo CLAs.

File: Bylaws.rst [hunk 2]
```diff
@@ -201,6 +201,8 @@ Article 22
 
 OCA whishes that all contributors of ideas, code, or documentation to the Association projects complete, sign, and submit (via postal mail, fax or email) an Individual Contributor License Agreement (CLA). The purpose of this agreement is to clearly define the terms under which intellectual property has been contributed to the Association and thereby allow the Association to defend the project should there be a legal dispute regarding the software at some future time. A signed CLA is required for every Delegate member and before an individual is given commit rights to an OCA project.
 
+**Note** The CLA signature process is a bit different from the process applied by Odoo S.A. Also the CLA itself is of course a different one. The OCA always requires written and signed documents and the OCA-Bot, upon commit, checks against a CLA flag in the backend of the OCA Odoo instance whether somebody has already signed the ICLA or is covered by an ECLA.
```
Comment (elicoidal) [inline Bylaws.rst]: for notes in rst:
`.. note::`

**Comment by pedrobaeza** [comment]
How is the state of this?

File: Bylaws.rst (hunk 2 above)
Comment (OSevangelist) [reply Bylaws.rst]: Hi @pedrobaeza hi @elicoidal i think the pull request may be declined as it seem favorable to put that note on the website. I will take care of this next week

**Comment by pedrobaeza** [comment]
OK then.

**Pull request closed without merging**